protocol/migrations/sqlite/1710264101_add_bio_to_contacts.up.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE contacts ADD COLUMN bio TEXT NOT NULL DEFAULT "";
```

Comment:
This could be an expensive migration, since the contact table is quite large.
Probably safer to allow NULLS (so it doesn't require the UPDATE below), and use `sql.NullString` when pulling the data from the database (`type Contact struct { Bio string}` is fine, but when you pull it from db, you store it in a NullString, and then 
`if bio.Valid` you set it (there's a few examples of this in the codebase), but ping me if it's not clear.

Reply:
Okay sounds good! I think I found some examples, I'll request re-review when I've pushed those changes 🙏